connect-button/src/main/java/com/ifttt/connect/ui/ButtonApiHelper.java

```diff
@@ -162,14 +172,10 @@ private Intent getIntentToApp(Context context, Connection connection, String ema
         return launchIntent;
     }
 
-    boolean shouldPresentCreateAccount(Context context) {
-        return shouldPresentEmail(context) && !accountFound;
-    }
-
```

Comment:
intentionally removed? Seems like we aren't using this anywhere.

Reply:
It's just moved to a different place.